odd_collector_gcp/adapters/gcs_delta/adapter.py

```diff
@@ -0,0 +1,40 @@
+from typing import Union
+
+from funcy import lmap, mapcat, partial
+from odd_collector_sdk.domain.adapter import BaseAdapter
+from odd_models.models import DataEntityList
+from oddrn_generator.generators import GCSGenerator
+
+from odd_collector_gcp.domain.plugin import GCSDeltaPlugin
+
+from .client import DeltaClient
+from .logger import logger
+from .mappers.delta_table import map_delta_table
+
+
+# TODO: Add tags
```

Comment:
What is tags?

Reply:
I wanted to ask the same thing because this TODO was mentioned in the s3_delta adapter and I left it there as a reminder.